rules/windows/process_creation/win_etw_trace_evasion.yml

```diff
@@ -19,13 +19,38 @@ logsource:
     product: windows
 detection:
     selection_clear_1:
-        CommandLine: '* cl */Trace*'
+        CommandLine|contains|all: 
+          - 'cl'
+          - '/Trace'
     selection_clear_2:
-        CommandLine: '* clear-log */Trace*'
+        CommandLine|contains|all: 
+          - 'clear-log'
+          - '/Trace'
     selection_disable_1:
-        CommandLine: '* sl* /e:false*'
+        CommandLine|contains|all: 
+          - 'sl'
+          - '/e:false'
     selection_disable_2:
-        CommandLine: '* set-log* /e:false*'
-    condition: selection_clear_1 or selection_clear_2 or selection_disable_1 or selection_disable_2
+        CommandLine|contains|all: 
+          - 'set-log'
+          - '/e:false'
+    selection_disable_3: #Autologger provider removal
+        Commandline|contains|all:
+          - 'Remove-EtwTraceProvider'
+          - 'EventLog-Microsoft-Windows-WMI-Activity-Trace'
+          - '{1418ef04-b0b4-4623-bf7e-d74ab47bbdaa}'
+    selection_disable_4: #Provider “Enable” property modification
+        Commandline|contains|all:
+          - 'Set-EtwTraceProvider'
+          - '{1418ef04-b0b4-4623-bf7e-d74ab47bbdaa}'
+          - 'EventLog-Microsoft-Windows-WMI-Activity-Trace'
+          - '0x11'
+    selection_disable_5: #ETW provider removal from a trace session
+        Commandline|contains|all:
+          - "logman" 
+          - "update trace" 
+          - "Trace --p"
```

Comment:
Hello @w0rk3r! Great job with the rule, it mostly looks great. There is only one tiny issue — the spaces here:

```
          - "update trace" 
          - "Trace --p"
```

Could you please remove them, splitting the strings into separate parts, like that:

```
          - "update"
          -  "trace" 
          - "Trace"
          - "--p"
```

I would ask the maintainers to merge the PR after that.
Thank you!

Reply:
Updated the rule, just kept one trace as it would match the two variants anyway, ty!

Reply:
awesome! thank you!